frontend/src/app/user/my-conferences/postpone-dialog/postpone-dialog.component.ts

```diff
@@ -0,0 +1,60 @@
+import {Component, Inject, OnInit} from '@angular/core';
+import {MAT_DIALOG_DATA, MatDialogRef} from '@angular/material/dialog';
+import {DialogData} from '../../my-submissions/recommendation-dialog/recommendation-dialog.component';
+import {Conference} from '../../shared/conference.model';
+import {ConferenceService} from '../../shared/conference.service';
+
+@Component({
+  selector: 'app-postpone-dialog',
+  templateUrl: './postpone-dialog.component.html',
+  styleUrls: ['./postpone-dialog.component.scss']
+})
+export class PostponeDialogComponent implements OnInit {
+  phase = 'IMPLEMENT BACKEND DUDE';
+  conference: Conference;
+
+  constructor(private dialogRef: MatDialogRef<PostponeDialogComponent>,
+              @Inject(MAT_DIALOG_DATA) public data: DialogData,
+              private conferenceService: ConferenceService) {
+  }
+
+  ngOnInit(): void {
+    const currentDate: Date = new Date();
+    // @ts-ignore
```

Comment:
create a new type for the dialog data that includes the conference. don't just ignore ts warnings :)